examples/test/Visualization_test.js

```diff
@@ -15,7 +15,7 @@ test('Chart visualization should render', async (t) => {
         .expect(chart.exists)
         .ok()
         .expect(chart.textContent)
-        .eql('Created with Highcharts 4.2.5Month/Year (Date)$ Total Sales$2,621,884$2,625,589$984,801Oct 2017Nov 2017Dec 20170M1M2M3M');
```

Comment:
What caused this change?

Reply:
Tested visualization has now static period filter.